### ISSUE TITLE
Whitepaper Things

### ISSUE THREAD

**Issue by DavidVorick** [NebulousLabs/Sia#3]
I'm basically using this as a place for notes.

We should mention in the whitepaper that using the blockchain to find hosts is completely optional. A centralized service (like a coin exchange, a forum, a private tracker) is a perfectly acceptible way to find hosts that you are interested in.

Generally though we like having all of the hosts listed in one place because then the random file placement works better.
# 

I am worried about a few things. Simplicity is nice but also scary. With the new system, there's nothing forcing hosts to be available to everyone. It felt more rigorously powerful to force hosts to accept random files, and it felt more rigorously powerful to force files upon random hosts.

It's more powerful because the big players can't leave the system. Super reliable hosts couldn't go off to the side and collect more expensive fees, scaring less reliable hosts away. Also, hosts had no incentive to be "super reliable" if it was significantly more expensive than being "mostly reliable". I'm afraid that people are going to be suboptimally trusting more expensive "super reliable" nodes when in fact the safest behavior is to buy 2 or 3 slots on less reliable nodes than a single larger slot on a super reliable node.

The old Sia didn't have any room for people changing that.

Basically we're trusting that our control of the client will be sufficient to prevent people from doing largely suboptimal things.

I guess that's also an advantage to us though. If our client can produce statistics that's substantially more reliable, cheaper, and faster than other clients, nobody is going to use a competing service. We have a way to stand out without losing customer share to other coins, because our super awesome client won't store on other coins.

Or something. Maybe it will, but only after we take a fee for ourselves again. lol.

It's just nerves though. I think that by being really good we'll be able to stay well ahead of the competition. But that requires actually being really good.
